todo-list/src/components/TodoItem.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <li>
+    <div class="todo__row">
+      <v-checkbox
+        color="indigo"
+        hide-details
+        v-show="editableTodo !== todo.id"
+        v-model="todo.completed"
+        @click="completeTodo({ id: todo.id, isCompleted: todo.completed })"
```

Comment:
Use `change` or `input` events for it. Checkbox can be toggled not only via click